es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 **1. Unsupervised Learning cheatsheet**
 
-&#10230;
+&#10230; Aprendizaje no Supervisado
```

Comment:
Translate also "cheatsheet" as "_hoja de referencia_".

Reply:
could it be "Hoja de Referencia de Aprendizaje no Supervisado"?